lisp/forge-topic.el

```diff
@@ -210,6 +210,14 @@ The following %-sequences are supported:
   (forge--format-url (forge-get-repository topic) slot
                      `(,@spec (?i . ,(oref topic number)))))
 
+(cl-defmethod forge-visit ((topic forge-topic))
+  (let ((magit-generate-buffer-name-function 'forge-topic-buffer-name))
+    (magit-mode-setup-internal #'forge-topic-mode (list topic) t)))
+
+(cl-defmethod forge-visit :after
+  ((topic forge-topic))
```

Comment:
Why put that on a separate line? I often collapse everything using `outline-minor-mode` (with the help of my `bicycle`) and then that is inconvenient.

Reply:
The indentation enforced by emacs is horrible when `:after` and friends are used; uses of this feature in core suggest this is the preferred solution. Personally I think the indentation logic should be fixed, but that is not my cup of tea.

Reply:
Indentation ended up being fixed after all 😄